cppcache/integration/test/PartitionRegionOpsTest.cpp

```diff
@@ -89,19 +89,36 @@ std::shared_ptr<Region> setupRegion(Cache& cache,
 }
 
 void putEntries(std::shared_ptr<Region> region, int numEntries,
-                int offsetForValue) {
+                int offsetForValue, bool isAllOp) {
```

Comment:
Boolean parameters generally are a code smell (https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3), and I don't think these are doing you any favors.  Have a look at refactoring to remove isAllOp altogether, I think it'll be better.

Reply:
Yah, I think this aligns with echobravopapa's suggestions above. Please have a look at my refactor suggestion above and let me know if that works.

Reply:
The refactor has removed the isAllOp parameter in favor of simpler functions.